src/FeedbackForm/FeedbackForm.js

```diff
@@ -150,7 +150,7 @@ export default function FeedbackForm({
                     <span>I like Something</span>
                   </Typography>
                   <Typography sx={{ mb: 1 }} color="text.primary">
-                    We like to hear what we're doing right.
+                    {"We like to hear what we're doing right."}
```

Comment:
Why did you make this change? I don't see anything wrong with the old version. I see the change made in a few places, not just here.

Reply:
To fix eslint error regarding special characters: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md

Reply:
As discussed, I disabled this eslint check and reverted the code.